asm/istio/options/cni-managed.yaml

```diff
@@ -0,0 +1,232 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
```

Comment:
Can we add some comment on how this file is generated?

Reply:
I think it was originally generated w/ `manifest generate`, but I don't think that we're going to continue generating it like that. Do you still think it's worth putting that in the file? Or can we just note it in the PR/commit for posterity?

Reply:
OK, yeah, as long as we are not going to repeat the process I think it is fine to just merge as is. Note in PR comment would be good enough.